src/sentry/testutils/cases.py

```diff
@@ -1809,6 +1809,7 @@ def setUp(self):
     def add_create_repository_responses(self, repository_config):
         raise NotImplementedError(f"implement for {type(self).__module__}.{type(self).__name__}")
 
+    @exempt_from_silo_limits()
```

Comment:
nit: should maybe be a factory?

Reply:
`sentry.testutils.factories.Factories.create_repo` already exists. This appears to be doing something more specialized, and may have been intended to exercise the `"sentry-api-0-organization-repositories"` endpoint.